helm-charts/splunk-otel-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: splunk-otel-collector
-version: 0.43.4
+version: 0.43.5
```

Comment:
If you want to release it right away, please add a section to CHANGELOG and run `make render`

Reply:
Feel free to keep the current version. we will release it after